packages/TagLabel/TagLabel.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <span class="tag" :class="type">
+    <slot />
+  </span>
+</template>
+
+<script>
+export default {
+  name: 'TagLabel',
+  props: {
+    type: { type: String }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@mesg-components/theme/_variables';
+
+.tag {
+  display: flex;
```

Comment:
Tag is full width which is not good, it's should be just the size of the text. Probably remove the display flex and align-items and/or add a display inline block

Reply:
Fixed